version/rhpam/version.go

```diff
@@ -16,11 +16,11 @@ package rhpam
 
 var (
 	// Version - current version
-	Version = "8.0.4"
+	Version = "8.0.5"
 	// PriorVersion product version
 	PriorVersion = "8.0.4"
 	// CsvVersion - csv release
-	CsvVersion = Version + "-2"
+	CsvVersion = Version + "-1"
 	// CsvPriorVersion - prior csv release
-	CsvPriorVersion = PriorVersion + "-1"
+	CsvPriorVersion = PriorVersion + "-3"
```

Comment:
I believe here should be "-2", no? Can you please double-check?

IIRC, We released only 2 versions of the Kogito bundle for 8.0.4